src/compiler/diagnosticMessages.json

```diff
@@ -4866,6 +4838,64 @@
         "code": 6242
     },
 
+    "Modules": {

```

Comment:
Does this need to start with 6243 instead of 6250 ..  i dont think it needs to be different order start.. if needs to be then 6300 or some such larger number might be better giving enough room in between these slots for additions?

Reply:
Need: nope - I can change it, makes sense to me 👍🏻